opencti-platform/opencti-graphql/src/database/file-search.js

```diff
@@ -132,16 +132,26 @@ const buildFilesSearchResult = (data, first, searchAfter, connectionFormat = tru
   }
   return convertedHits;
 };
+const decodeSearch = (search) => {
+  let decodedSearch;
+  try {
+    decodedSearch = decodeURIComponent(search).trim();
+  } catch (e) {
+    decodedSearch = search.trim();
+  }
+  return decodedSearch;
+};
 const elBuildSearchFilesQueryBody = async (context, user, options = {}) => {
   const { search = null, fileIds = [], entityIds = [] } = options; // search options
   const { includeAuthorities = false } = options;
   const dataRestrictions = await buildDataRestrictions(context, user, { includeAuthorities });
   const must = [...dataRestrictions.must];
   const mustNot = [...dataRestrictions.must_not];
   if (search) {
+    const decodedSearch = decodeSearch(search);
```

Comment:
I think specialElasticCharsEscape should be used

Reply:
why do we need `specialElasticCharsEscape` ?

Reply:
To prevent query to crash because user is not aware of elasticsearch query syntax

Reply:
Then we can use simple_query_string which will work with special characters even with syntax errors : https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html

Reply:
yes I tried already and there is no error